script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -72,7 +72,7 @@ I honestly completely forgot that my left hand is a wreck.
 -- Page 364, Offset 36350.
 -- あ、でも死徒化しているなら秘蹟は逆効果だから、えっと、この場合はどうすれば―――」
 Oh, but you're turning into a Dead Apostle, so a sacrament will have the opposite effect...
- uhh, what should I do―――"
+ uhh, what should I dooo―――"
```

Comment:
🤔 wait, what was this change for

Reply:
Guessing to make the VA sound more panicked?

Reply:
I thought it maybe had something to do with the voice, but it's quite the opposite. She sounds composed, and voices it briefly. I'd say revert?